What’s up in Vietnam.md

```diff
@@ -65,58 +66,58 @@ UNION Col1, Null as Col2, Col3 FROM heating_devices.
 
-This text is a bit naive too. I’m not forcing you to use more difficult tool to achieve a kind of karma. Relational databases and ORMs are advanced tools and I’m using it often privately. Just think at first a little bit if chosen technology fits your requirements. Relational databases and ORMs are not the only right way, it shouldn’t be even default way. It should be a decision.
+This text is a bit naive too. I’m not forcing you to use more difficult tool to achieve a kind of developers' nirvana. Relational databases and ORMs are advanced tools and I’m using them often. Firstly, just think a little bit if chosen technology fits your requirements. Relational databases and ORMs are not the only right way, it shouldn’t be even a default way. It should be a careful decision to use it. 
 
 ## So … what changed since Vietnam War and what are we yet to solve in ORMs?
 So, 12 years have passed.
-1. Some problems have evolved. Joins are much faster now thanks to solutions like ClusterixDB. There are newly built-in mechanisms in relational databases. We can store documents in PostgreSQL database. Although it’s still not the best idea to create DML query with dozens of joins, but modern databases are optimized for less complex queries.
+1. Some problems have evolved. Joins are much faster now thanks to solutions like ClusterixDB. There are new built-in mechanisms in relational databases. We can store documents in PostgreSQL database. It’s still not the best idea to create DML query with dozens of joins, but modern databases are optimized for less complex queries.
 2. Over time, composition started winning over inheritance in OOP. Mapping composition to RDBS is easier thanks to its relational origin.
-3. Using the same database by two applications is seen as a bad practice. This and others policy problems mentioned by Ted Neward are a little outdated. But on the other hand, many services are starting cooperatively. Updating such services is a quite hard to achieve. There is a time where one service needs to work with two versions of a database. Forcing ORM framework to work with two different versions of the database is almost impossible in clear and easy to achieve way.
+3. Using the same database by two applications is seen as a bad practice. This and others policy problems mentioned by Ted Neward are a little outdated. But on the other hand, many services are starting to use databases cooperatively. Updating such services is a quite hard. There is a time where one service needs to work with two versions of a database. Forcing ORM framework to work with two different versions of the database is almost impossible in clear and easy to achieve way.

```

Comment:
It lost some sense. My proposition:
"But on the other hand, there are many distributed applications with dozens of services. Updating such services with a database migration is quite hard to achieve."

Reply:
I like your version more, maybe say "is quite hard task" instead "hard to achieve".

Reply:
done